app/src/features/docs/lib/api.ts

```diff
@@ -0,0 +1,109 @@
+import HTTP, { PackagePreview } from "../../../utils/http";
+import { RecentPackage, RecentPackagesResponse as APIRecentPackagesResponse } from "../../dashboard/hooks/useFetchRecentPackages";
```

Comment:
nit: explicit import of types vs objects:

```suggestion
import HTTP from "../../../utils/http";
import type { PackagePreview } from "../../../utils/http";
import type { RecentPackage, RecentPackagesResponse as APIRecentPackagesResponse } from "../../dashboard/hooks/useFetchRecentPackages";
```